clang/lib/AST/Interp/Compiler.cpp

```diff
@@ -3593,8 +3593,14 @@ VarCreationState Compiler<Emitter>::visitDecl(const VarDecl *VD) {
   if (R.notCreated())
     return R;
 
-  if (R)
-    return true;
+  if (R) {
```

Comment:
This change just follow current ExprConstant's behavior, but it's no value for this issue, should we remove it?

Reply:
Yes, remove it please.

Reply:
Removed.